exercise/templates/exercise/_exercise_info.html

```diff
@@ -6,7 +6,7 @@
     {% if exercise.category.confirm_the_level %}
     <p>{% translate "CURRENT_STATUS" %}</p>
     <p class="exercise-info-points">
-      {% points_badge summary %}
+      {% points_badge summary is_revealed=feedback_revealed %}
```

Comment:
The default value for the `is_revealed` parameter of `points_badge` (and `points_progress`) is `None`. If `None` is passed, the reveal rule will be evaluated. If `True` or `False` is passed, the value is used directly and the reveal rule will not be evaluated.

Reply:
I meant how the template variables behave. If the template variable is not defined, I guess the Python function will use the default None value. The template language is not Python and in theory, it could behave in a surprising way when passing parameters from non-existing variables.

Reply:
I tested this. The default value for template tag parameters that are not passed is None.